### PR TITLE
fix(rdb-acl): tags, ACL description attribute and documentation

### PR DESCRIPTION
**Pull request by Monitob**

### CODE REVIEW

File: docs/resources/rdb_acl.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The following arguments are supported:
 The `acl_rules` block supports:
 
 - `ip` - (Required) The ip range to whitelist in [CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation)
-- `description` - (Optional) A simple text describing this rule
+- `description` - (Optional) A simple text describing this rule. Default description: `IP allowed`
```
Comment (remyleone) [inline docs/resources/rdb_acl.md:39]:
```suggestion
- `description` - (Optional) A text describing this rule. Default description: `IP allowed`
```

File: scaleway/resource_rdb_instance.go [hunk 2]
```diff
@@ -442,7 +449,7 @@ func resourceScalewayRdbInstanceUpdate(ctx context.Context, d *schema.ResourceDa
 	if d.HasChange("backup_schedule_retention") {
 		req.BackupScheduleRetention = scw.Uint32Ptr(uint32(d.Get("backup_schedule_retention").(int)))
 	}
-	if d.HasChange("tags") {
+	if d.HasChangeExcept("tags") {
```
Comment (remyleone) [inline scaleway/resource_rdb_instance.go:452]: Why this change is required?

Comment (Monitob) [reply scaleway/resource_rdb_instance.go:452]: Unfortunately not. I'll add a new test and keep HasChange

**Review by remyleone — APPROVED** [review]
